File: DisplaFrontend/src/app/model/sobre.ts
import { Cliente } from './cliente';
import { Usuario } from './usuario';

export interface Sobre {
    Id: number,
    Numero: number,
    Entregas: number,
    Observaciones: string,
    Fecha: Date,
    IdCliente: number,
    IdClienteNavigation: Cliente,
    IdUsuario: number,
    IdUsuarioNavigation: Usuario
}

File: DisplaBackend/DisplaBackend/Models/Sobre.cs
using System;
using System.Collections.Generic;

namespace DisplaBackend.Models
{
    public partial class Sobre
    {
        public int Id { get; set; }
        public int Numero { get; set; }
        public int Entregas { get; set; }
        public string Observaciones { get; set; }
        public DateTime Fecha { get; set; }
        public int IdCliente { get; set; }
        public int IdUsuario { get; set; }

        public virtual Cliente IdClienteNavigation { get; set; }
        public virtual AspNetUsers IdUsuarioNavigation { get; set; }
    }
}
